src/node/entities.h

```diff
@@ -81,6 +81,8 @@ namespace ccf
     static constexpr auto GOV_HISTORY = "public:ccf.gov.governance.history";
     static constexpr auto SERVICE = "public:ccf.gov.service";
     static constexpr auto SHARES = "public:ccf.gov.shares";
+    static constexpr auto ENCRYPTED_LEDGER_SECRETS =
```

Comment:
This is a new map, can we have it be documented? Also, this feels much more like a internal rather than a gov table.

Reply:
If this map is internal (as your other comment suggests, which I also believe is right), do we still need to document it?

Reply:
As discussed, this needs documenting as part of #230